pages/blog/[blogpostId].jsx

```diff
@@ -17,17 +19,20 @@ export default function SingularPost() {
   }, [setActivePost, blogpostId]);
 
   return (
-    <article>
-      {(activePost === null || activePost === undefined) && (
-        <Typography
-          sx={{ fontSize: 32, fontWeight: 700, textAlign: 'center' }}
-          variant="h3"
-        >
-          {activePost === null ? 'Loading…' : 'Post not found!'}
-        </Typography>
+    <>
+      {isHydrated && (
+        <section>
+          {(activePost === null || activePost === undefined) && (
```

Comment:
```suggestion
          {!activePost && (
```

Seems like this would do the same, considering that your IDs are never the number `0` or an empty string.